contents/julia_101.md

```diff
@@ -2037,6 +2037,43 @@ sym = Symbol(s)
 One simple benefit of symbols is that you have to type one character less, that is, `:some_text` versus `"some text"`.
 We use `Symbol`s a lot in data manipulations with the `DataFrames.jl` package (@sec:dataframes) and data visualizations (@sec:plots and -@sec:makie).
 
+### Splat Operator {#sec:splat}
+
+In Julia we have the "splat" operator `...` which is mainly used in function calls as a **sequence of arguments**.
+We will occasionally use splatting in some function calls in **data manipulation** and **data visualization** chapters.
+
+The `...` operator takes a collection (often an array, vector, tuple or range) and converts into a sequence of arguments.
+The `...` is included after the collection that we want to "splat" into a sequence of arguments.
+For example, if we call a function on a range followed by `...`, it is the same as calling a function on all arguments of a collection.
```

Comment:
3blue1brown advises to give an example first and then discuss the nitty gritty details. I think this would be suitable here, because I'm afraid that I don't understand this sentence 😬

Reply:
> Slurping should also be added now or later ([docs.julialang.org/en/v1/manual/faq/#The-two-uses-of-the-...-operator:-slurping-and-splatting](https://docs.julialang.org/en/v1/manual/faq/#The-two-uses-of-the-...-operator:-slurping-and-splatting))

Do we really need to cover slurping? the two cases that `...` appeared were splatting inside function calls in both dataframes and datavis chapters...

Will our user need to know the two cases slurping/splatting?

I am asking this because I want to justify every new increase of complexity.

Reply:
> 3blue1brown advises to give an example first and then discuss the nitty gritty details. I think this would be suitable here, because I'm afraid that I don't understand this sentence

I will try to do the example first then and see if it makes it clearer.

Reply:
Oh yes, you are very right. Slurping is not as important!